examples/stencil/src/lib.rs

```diff
@@ -131,22 +133,24 @@ fn assert_data_eq(a: &Data, b: &Data) {
             for x in 0..a.n.0 {
                 let idx = (x + y * a.n.1 + z * a.n.1 * a.n.0) as usize;
 
-                assert_eq!(
-                    a.vsq[idx], b.vsq[idx],
+                const EPSILON: f32 = 1E-4;
```

Comment:
Is this the smallest absolute epsilon that works?

Reply:
Yes, at least on my computer.